client/src/components/Related/Carousel.jsx

```diff
@@ -5,9 +5,20 @@ import ForwardArrow from './ForwardArrow';
 import ReverseArrow from './ReverseArrow';
 
 const Carousel = (props) => {
-  const { data, btn } = props;
+  const { onAddOutfitClick, data, btn } = props;
   return(
     <div data-testid="carousel">
+      {/* Add Outfit Button Card */}
+      {
+        btn === 'compare'
+        ?
+        <></>
```

Comment:
I this could just be "null"

Reply:
changed it locally, it will be in my next commit
{
        btn !== 'compare' &&
        <Card
          onAddOutfitClick={onAddOutfitClick}
          addOutfit='addOutfit'
        />
}